### PR TITLE
Exchange capabilities - rewrite for better visibility

### PR DESCRIPTION
**Pull request by ttodua**
I've been trying to still make the capabilities functionality more clear and sorted by priorities & common unification order.

Also, as most npm commands are for devs mostly, the exchange-capabilities (to help devs & see most of CCXT functionalities in one output) command is a handy one for either core-devs and non-core-devs to see ccxt capabilities easily.  So, I think it's helpful to be in package.json shorthands too.

Please check it's output and let me know if you like that way, or if you suggest any edition.

### CODE REVIEW

File: examples/js/exchange-capabilities.js [hunk 1]
```diff
@@ -20,109 +21,137 @@ const isWindows = process.platform == 'win32' // fix for windows, as it doesn't
...
+        'fetchOrder',
+        'fetchOrders',
+        'fetchOpenOrders',
+        'fetchClosedOrders',
+        'fetchOrderTrades',
+        'fetchMyTrades',
+        'cancelAllOrders',
+        'fetchPositions',
+    ]};
+    let uncommonEndpoints = { 'uncommon': [
```
[the remark this answers is not in the document]
Comment (ttodua) [reply examples/js/exchange-capabilities.js:66]: The reason I've just placed them in common/uncommon was the amount of exchanges adoption. where only few of exchanges (i.e. 10 from 100) had this-or-that functionality, i've placed them under uncommon. However, as you think, let it be  that way.
So, should I update, or you are about to do that?

[the remark this answers is not in the document]
Comment (frosty00) [reply examples/js/exchange-capabilities.js:66]: we should probably keep them in one array, btw we can also use the same api as defined here to automatically parse the has array

[the remark this answers is not in the document]
Comment (ttodua) [reply examples/js/exchange-capabilities.js:66]: @kroitor @frosty00 so, should I close this PR?

**Comment by ttodua** [comment]
Closed, per your advise.

**Pull request closed without merging**